package/MDAnalysis/units.py

```diff
@@ -343,16 +343,14 @@ def convert(x, u1, u2):
     try:
         ut1 = unit_types[u1]
     except KeyError:
-        raise ValueError(("unit '{0}' not recognized.\n" +
-                          len("ValueError: ")*[" "] +
-                          "It must be one of {1}.").format(u1, unit_types))
+        raise ValueError("unit '{0}' not recognized.\n"
+                          "It must be one of {1}.".format(u1, ", ".join(unit_types)))
```

Comment:
@orbeckst This is what I thought.

@BartBruininks Could you remove all the keys that contain ` b'\xe2\x84\xab'` in the `*Unit_factor` dicts? It should fix your tests on python 3.